client/src/components/ExtensionContext.ts

```diff
@@ -0,0 +1,24 @@
+import { ExtensionContext } from "vscode";
```

Comment:
This seems like a persistence layer for a workspace. I'm wondering if our function names should be more inline with the abstract concept or persistence rather than the specific implementation. Perhaps this should expose `get` and `persist` or `persistValue`/`getPersistedValue`. Thoughts?